### DISCUSSION TITLE
how to add caption?

### DISCUSSION THREAD

**Discussion by zulmarij** [awcodes/filament-tiptap-editor#462] [Q&A]
I use media actions from curator.
How to add captions when adding media?

**Comment by awcodes** [comment]
Currently Tiptap editor plugin doesn't have a figure component so there's no way to add a caption, but if the media item in curator has a title set then it will be added to the img.

**Reply by TonsiTT** [reply]
could this be used? 
https://tiptap.dev/docs/examples/experiments/figure

**Reply by awcodes** [reply]
It could, but it'll take some work on your part to make it work with curator or even Tiptap editor. I'm not willing to include anything in the plugin that isn't officially released though.
